eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -144,6 +144,16 @@ function GeneratePRMatrixForBatch {
       }
     }
 
+    if ($matrixConfig.PSObject.Properties['PRBatch']) {
+      # if we are doing a PR Batch, we need to just add the matrix items directly to the OverallResult
+      # as the users have explicitly disabled PR batching for this matrix.
+      if (!$matrixConfig.PRBatch) {
+        Write-Host "The matrix config $($matrixConfig.Path) has PRBatch set to false, the matrix members will be directly added without batching by $PRMatrixSetting."
```

Comment:
Does `.Path` point to the json file? Could be misleading since it's in the yaml.

Reply:
@benbp the Path setting is part of the matrix configuration that identifies the where the json file is. @scbedd, maybe the name + path in the message?
```
AdditionalMatrixConfigs:
  - Name: version_overrides_tests
    Path: sdk/core/version-overrides-matrix.json
    Selection: all
    GenerateVMJobs: true
```